### ISSUE TITLE
Depend on quarkus-minio-native instead of quarkus-minio to be able to create Minio clients programmatically

### ISSUE THREAD

**Issue by zbendhiba** [apache/camel-quarkus#4328]
I tried to upgrade the minio version quickly to 2.10.2 version, but integration tests are failing.
First, minio has now devServices so this configuration needs to be added to the  application.properties file : `quarkus.minio.devservices.enabled = false`.
Second there’s now a mandatory minio.url, which may not be compliant with camel-quarkus. Here’s a part of the error :
```
Caused by: io.quarkus.runtime.configuration.ConfigurationException: "quarkus.minio.url" is mandatory and must be a valid url
```

**Comment by ppalaga** [comment]
I have filed https://github.com/quarkiverse/quarkus-minio/issues/178

**Comment by ppalaga** [comment]
Solved in quarkus-minio:2.10.3, but the change is backwards incompatible for us. We need to depend on quarkus-minio-native, which I did in https://github.com/apache/camel-quarkus/pull/4340
